### ISSUE TITLE
Default room script crashes

### ISSUE THREAD

**Issue by clarkjoe** [Kawa-oneechan/SCICompanion#23]
SCICompanion version: 3.2.4

When a new room script is debugged and the default Ego is moved, the room crashes with the following message:

![Screenshot 2023-08-13 124815](https://github.com/Kawa-oneechan/SCICompanion/assets/31975090/4ee47804-c842-47ca-9707-98f3ea2411d7)

I don't know the nuts of bolts of the failure, but I do know why it crashes. The default new room script, calls the `&getPolygon` procedure (`(gRoom addObstacle: (&getpoly {}))`). This fetches the "empty string" polygon. However, a new pic has zero polygons by default. So the new room script tries to access a nonexistent polygon; thus, resulting in the crash. Correct or improve this explanation if needed.

The SCI compiler emits the following debug statement when accessing a nonexistent polygon: `Unknown polygon name  in &getpoly`. This message is helpful, but the crash is frustrating when wanting to simply create a new room script and pic.

A possible solution is to comment out the `getpoly` procedure by default. A comment can be included to inform the user to uncomment the line once one or more polygon is added to the pic.

Furthermore, after the crash, when starting the game from the beginning, the following bug appears on the title screen:

![Screenshot 2023-08-13 124838](https://github.com/Kawa-oneechan/SCICompanion/assets/31975090/052bacc2-f1dc-465e-a3ff-180ef970bb21)

I do not know what happens for these strange characters to appear in the title menu. Clicking on any of the options results in the following error:

![Screenshot 2023-08-13 124923](https://github.com/Kawa-oneechan/SCICompanion/assets/31975090/3b364224-ccdc-4ab1-ab20-f7cc9d041c8e)

**Comment by Kawa-oneechan** [comment]
With the new nightly out, you should be able to confirm if things work.

**Comment by clarkjoe** [comment]
I pulled your changes locally and built and ran it. The new script includes the changes and a new room does not crash on Ego move.

When I downloaded and extracted the night build (`SCICompanion-Kawa-Win32-2023-08-15`), I get the following error when running the executable:

![Screenshot 2023-08-15 082001](https://github.com/Kawa-oneechan/SCICompanion/assets/31975090/7ae57e97-760f-4bb0-bf6b-dd0c1022c2a8)

From the dialog, it seems I have to find an already working SCICompanion release directory and replace the SCICompanion.exe with the new executable from the new release. Is that really the way to run the nightly releases? I would think the nightly release zip would contain everything to run SCICompanion on its own.
